### PR TITLE
Enabled properties

### PR DESCRIPTION
**Pull request by roigcarlo**
This PR should allow to use materials both in DEM and CFD wrappers.

@khiltunen I expect this to fail right now due to flakes, but it should allow you to modify the parameters we talk last Friday.

I will upload the example with materials here shortly, In the meantime if you wanna take a look...

### CODE REVIEW

File: example_scripts/coupling_no_mpda/MainCouplingNoMdpaParticles.py [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+    # materialData[CUBA.DEM_DISCONTINUUM_CONSTITUTIVE_LAW_NAME] = "DEM_D_Hertz_viscous_Coulomb"
+    # materialData[CUBA.CONTACT_TAU_ZERO] = 25
+    # materialData[CUBA.CONTACT_SIGMA_MIN] = 5
+    # materialData[CUBA.CONTACT_INTERNAL_FRICC] = 1
+
+    material.data = materialData
+
+    particlesData = particles.data
+    particlesData[CUBA.MATERIAL] = material.name
+    particles.data = particlesData
```
Comment (roigcarlo) [inline example_scripts/coupling_no_mpda/MainCouplingNoMdpaParticles.py:78]: @khiltunen This is the way to change the properties.
As you can see not all of them are enabled, so if they have no entry point in the CUBA keywords they will use the default value.

The wrapper is prepared so as long as soon as they can be stored in CUBA they can be enabled just adding its name in the wrapper.

If some of the properties commented are needed for your simulations, pleas tell me so we can ping someone who can update the oficial CUBA keyword list ( or make a PR requesting the changes in the simphony-metadata repo)

**Comment by khiltunen** [comment]
Using OpenFOAM coupling and OpenFOAM mesh i get this error

simPRO.run()
  File "/usr/local/lib/python2.7/dist-packages/simphony-0.7.0.dev0-py2.7.egg/simphony/cuds/simulation.py", line 37, in run
    self._wrapper.run(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/simkratos-0.4.0.dev0-py2.7.egg/simkratos/PROJECT/kratos_PROJECT_wrapper.py", line 378, in run
    group, self.condition_type
  File "/usr/local/lib/python2.7/dist-packages/simkratos-0.4.0.dev0-py2.7.egg/simkratos/kratosWrapper.py", line 677, in importKratosConditions
    self.condition_properties
AttributeError: 'PROJECTWrapper' object has no attribute 'condition_properties'

**Comment by roigcarlo** [comment]
@khiltunen Should be fixed. I am about to write an issue requesting the new CUBA, any particular one that is of special interest?

**Comment by roigcarlo** [comment]
@khiltunen This should fix the problem with the coords. If particles still have the same position please download Kratos again, I am not 100% sure the last release had the changes so I just create another release, but probably should be fine as is.

**Comment by roigcarlo** [comment]
@khiltunen The problem with the dt should be solved with the lasts commits

**Pull request merged**